openedx/core/djangoapps/course_roles/docs/decisions/0001-course_level_roles.rst

```diff
@@ -0,0 +1,89 @@
+1. Course Level Roles
+######################
+
+
+Status
+******
+
+**Provisional** *2023-09-13*
+
+The status will be updated to *Accepted* upong completion of reimplementation. Related work:
+
+- `Course Roles Setup <https://2u-internal.atlassian.net/browse/ROLES-2>`
+
+
+Context
+*******
+
+There is currently no single functionality being used by the majority of the codebases that are a part of Open edX
```

Comment:
For this ADR, let's keep the focus on users (remove groups of users from descriptions in all sections of the ADR).